package.json

```
@@ -15,7 +15,7 @@
     "@guardian/automat-modules": "^0.3.7",
     "@guardian/braze-components": "0.0.12",
     "@guardian/commercial-core": "^0.9.0",
-    "@guardian/consent-management-platform": "6.4.0",
+    "@guardian/consent-management-platform": "7.0.3",
```

[The future is now](https://github.com/guardian/consent-management-platform/releases/tag/v7.0.4)!

```suggestion
    "@guardian/consent-management-platform": "7.0.4",
```